src/content/en/2020/cms.md

```diff
@@ -23,3 +22,364 @@ published: 2020-12-09T00:00:00.000Z
+
+One of the main ways a CMS can influence its page load size is by supporting and encouraging the usage of more efficient formats. Images are behind only video in their contribution to page weight. 
+
+{{ figure_markup(
+  image="cms-median-cms-kb-per-resource-type.png",
+  caption="Median CMS KB per resource type.",
+  description="Bar chart showing the median resource size in KB of each resource type for CMSs. Audio 14 KB on desktop and 10 KB on mobile, CSS is 102 KB on desktop and 98 on mobile, Fonts are 152 KB on desktop and 125 on mobile, Images are 1,273 KB on desktop and 1,280 on mobile, Video is 1,665 KB on desktop and 2,139 KB on mobile and XML is 1 KB for both desktop and mobile.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTkUxfuK-FCn3_IYDJiEsKDmdmyKb0TSEhG8dFc0XBIXej1NO2uUOmd-9NCbWuh-MZ3xzMhK_kNT-4u/pubchart?oid=74536341&format=interactive",
+  sheets_gid="1846606749",
+  sql_file="third_party_bytes_and_requests_on_cmss.sql"
+  )
+}}
+
+Video contributes a larger percentage per resource type here. Making video more efficient is an interesting area for future research. Here our focus is on images. Popular image formats are JPG, PNG, GIF, SVG, WebP, and ICO. Of these, [WebP is the most efficient in most situations](https://developers.google.com/speed/webp/), with WebP lossless images [26% smaller](https://developers.google.com/speed/webp/docs/webp_lossless_alpha_study#results) than equivalent PNGs and [25-34% smaller](https://developers.google.com/speed/webp/docs/webp_study) than comparable JPGs. We see, however, that WebP is the second least popular image format across all CMS pages:
+
+{{ figure_markup(
+  image="cms-popularity-of-image-formats.png",
+  caption="Popularity of image formats.",
+  description="Bar chart showing the percentage of images by type on CMSs. Desktop follows mobile usage closely. jpg is 33% of image formats on both desktop and mobile, unknown is 25% for desktop and 26% for mobile, png is 25% for desktop and 23% for mobile, gif is 13% for both, svg is 2% for both, webp 1% for both as is ico.",
+  chart_url="https://docs.google.com/spreadsheets/d/e/2PACX-1vTkUxfuK-FCn3_IYDJiEsKDmdmyKb0TSEhG8dFc0XBIXej1NO2uUOmd-9NCbWuh-MZ3xzMhK_kNT-4u/pubchart?oid=2011368432&format=interactive",
+  sheets_gid="748745523",
+  sql_file="adoption_of_image_formats_in_cmss.sql"
+  )
+}}
+
+Of the top 5 CMSs, only Wix automatically converts and serves images in the WebP format. WordPress, Drupal, and Joomla support WebP with extensions, whilst at the time of writing Squarespace does not support WebP.
+
+As we saw earlier, Wix had the lowest proportion of sites with a “good” LCP. We can make a reasonable link between page weight and page load time, so clearly automatically converting to WebP is not a solution in isolation to page weights and thus CO2 emissions. It is, however, a more efficient format and improved native support for the format by the most popular CMSs would be beneficial.
```

Comment:
Wix has bigger issues with SPAs. I'm not sure that we can draw the connection between LCP and WebP.

Reply:
what do you think for this one then? Does "we can make a reasonable link" need to come out?

Reply:
My suggestion would be to acknowledge the unknowns, for example "While we know that Wix is making efficient use of image bytes in WebP, there are clearly other issues affecting its LCP performance beyond image formats that we aren't controlling for here."

Reply:
sure, done!